### PR TITLE
fix: linting errors on CI 

### LINKED ISSUE #844: Fix CI / backend lint
Small ticket to fix this linting issue: https://github.com/climateconnect/climateconnect/runs/4520216772?check_suite_focus=true

### PR DESCRIPTION
**Pull request by piperchester**
## Description
Fixes #844. 

## Before landing

1. PR has meaningful title
1. `yarn lint` passes
1. `yarn format` passes

### CODE REVIEW

File: backend/climateconnect_api/utility/email_setup.py [hunk 1]
```diff
@@ -215,8 +215,9 @@ def send_feedback_email(email, message, send_response):
         ]
     }
     print(data)
+
     try:
-        mail = mailjet_send_api.send.create(data=data)
+        mailjet_send_api.send.create(data=data)
```
Comment (piperchester) [inline backend/climateconnect_api/utility/email_setup.py:220]: Removed unused reference here.

File: backend/climateconnect_api/views/settings_views.py [hunk 2]
```diff
@@ -25,94 +27,132 @@ def get(self, request):
...
             return Response(
-                {'message':
-                    _('User profile not found.')
-                    + " " +
-                    _('Contact contact@climateconnect.earth if you repeatedly experience problems.')
-                 },
-                status=status.HTTP_400_BAD_REQUEST)
+                {
+                    "message": (
+                        "User profile not found. Contact contact@climateconnect.earth if you repeatedly experience problems."
```
Comment (piperchester) [inline backend/climateconnect_api/views/settings_views.py:135]: Dropped these prefixed `_`.

**Review by piperchester — COMMENTED** [review]
I also ran Black for formatting fixes across this file.

**Review by positiveimpact — APPROVED** [review]
LGTM!

**Pull request merged**